R/server.R

```diff
@@ -48,6 +49,8 @@ server <- function(input, output, session) {
                       selected = "Uploaddata")
   })
   
+  callModule(networkServer, "network", session, significant)
```

Comment:
Instead of passing significant, could you instead pass `data_comparison$ComparisonResult` instead here?